### PR TITLE
Gets blockchain storage info.

### PR DESCRIPTION
**Pull request by NemTechCN**
Gets blockchain storage info.

### CODE REVIEW

**Comment by dgarcia360** [comment]
Hi @NemTechCN , the code looks nice! To follow the [API approach](https://nemtech.github.io/endpoints.html#operation/getServerInfo), I would prefer if you could move the command either to ``nem2-cli diagnostic`` or ``nem2-cli node`` instead of ``nem2-cli blockchain``. Thanks!

File: src/commands/blockchain/storage.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        super();
+    }
+
+    @metadata
+    execute(options: ProfileOptions) {
+        this.spinner.start();
+
+        const profile = this.getProfile(options);
+
+        const blockchainHttp = new BlockchainHttp(profile.url);
```
Comment (dgarcia360) [inline src/commands/blockchain/storage.ts]: BlockchainHttp is now BlockHttp

File: src/commands/blockchain/storage.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        blockchainHttp.getDiagnosticStorage().subscribe((storage) => {
+            this.spinner.stop(true);
+
+            console.log('numBlocks: ' + storage.numBlocks);
```
Comment (dgarcia360) [inline src/commands/blockchain/storage.ts]: I would put all the output text into a ``text`` variable. Then, write one single console.log(text) at the end.

**Comment by dgarcia360** [comment]
@NemTechCN This PR is almost valid, do you want to continue it? Thanks!

File: src/commands/blockchain/storage.ts [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ * limitations under the License.
+ *
+ */
+import chalk from 'chalk';
+import {command, metadata} from 'clime';
+import {BlockchainHttp} from 'nem2-sdk';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+
+@command({
+    description: 'Blockchain storage',
```
Comment (dgarcia360) [inline src/commands/blockchain/storage.ts]: Get blockchain storage info

**Comment by dgarcia360** [comment]
@evias I have fixed the command and did some reorganization with the latest. Please, check the PR is okay,  and I proceed to merge it.

File: src/commands/diagnostic/serverinfo.ts [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+ */
+import chalk from 'chalk';
+import {command, metadata} from 'clime';
+import {DiagnosticHttp} from 'nem2-sdk';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+
+@command({
+    description: 'Returns the version of the running rest component',
+})
+export default class extends ProfileCommand {
```
Comment (evias) [inline src/commands/diagnostic/serverinfo.ts]: command name should be `ServerInfo`

File: src/commands/diagnostic/storage.ts [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+ */
+import chalk from 'chalk';
+import {command, metadata} from 'clime';
+import {DiagnosticHttp} from 'nem2-sdk';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+
+@command({
+    description: 'Returns diagnostic information about the node storage',
+})
+export default class extends ProfileCommand {
```
Comment (evias) [inline src/commands/diagnostic/storage.ts]: command name should be `Storage`

**Pull request merged**